### PR TITLE
Replace FontBase with IFont

### PR DESCRIPTION
**Pull request by adrianstevens**

### CODE REVIEW

File: Source/Meadow.Foundation.Libraries_and_Frameworks/Graphics/MicroGraphics/Driver/Graphics.MicroGraphics/Fonts/FontBase.cs [hunk 1]
```diff
@@ -1,25 +1,17 @@
-﻿namespace Meadow.Foundation.Graphics
+﻿using System;
+
+namespace Meadow.Foundation.Graphics
 {
     /// <summary>
     ///     Abstract class for a Font.
     /// </summary>
-    public abstract class FontBase
+    [Obsolete("FontBase is depricated, use IFont instead")]
```
Comment (jorgedevs) [inline Source/Meadow.Foundation.Libraries_and_Frameworks/Graphics/MicroGraphics/Driver/Graphics.MicroGraphics/Fonts/FontBase.cs:8]: *deprecated, right? :)

File: Source/Meadow.Foundation.Peripherals/Displays.ePaper/Driver/Displays.ePaper/Drivers/IL91874.cs [hunk 2]
```diff
@@ -17,7 +17,7 @@ public class Il91874 : EpdColorBase
 
         protected override bool IsBlackInverted => true;
 
-        protected override bool IsColorInverted => true;
+        protected override bool IsColorInverted => false; //is this backwards??
```
Comment (jorgedevs) [inline Source/Meadow.Foundation.Peripherals/Displays.ePaper/Driver/Displays.ePaper/Drivers/IL91874.cs:20]: Can you verify this?

**Review by jorgedevs — APPROVED** [review]

File: Source/Meadow.Foundation.Libraries_and_Frameworks/Graphics/MicroGraphics/Driver/Graphics.MicroGraphics/Fonts/FontBase.cs (hunk 1 above)
Comment (adrianstevens) [reply Source/Meadow.Foundation.Libraries_and_Frameworks/Graphics/MicroGraphics/Driver/Graphics.MicroGraphics/Fonts/FontBase.cs:8]: *cough* ... yup

File: Source/Meadow.Foundation.Peripherals/Displays.ePaper/Driver/Displays.ePaper/Drivers/IL91874.cs (hunk 2 above)
Comment (adrianstevens) [reply Source/Meadow.Foundation.Peripherals/Displays.ePaper/Driver/Displays.ePaper/Drivers/IL91874.cs:20]: verified - I'll clean up the comment

**Pull request merged**